common/src/main/java/com/revenuecat/purchases/common/offlineentitlements/PurchasedProduct.kt

```diff
@@ -7,7 +7,6 @@ data class PurchasedProduct(
     val productIdentifier: String,
     val basePlanId: String?,
     val storeTransaction: StoreTransaction,
-    val isActive: Boolean,
```

Comment:
Now all these will be active, so I just removed this property.

Reply:
It was a very confusing property too